lib/withOnyx.js

```diff
@@ -121,14 +121,9 @@ export default function (mapOnyxToState) {
                     return;
                 }
 
-                const stateUpdate = {...this.tempState, loading: false};
-
-                // The state is set here manually, instead of using setState because setState is an async operation, meaning it might execute on the next tick,
-                // or at a later point in the microtask queue. That can lead to a race condition where
-                // setWithOnyxState is called before the state is actually set. This results in unreliable behavior when checking the loading state and has been mainly observed on fabric.
```

Comment:
Is all this stuff no longer true? Can't say this old comment really makes any sense to me tbh.

Reply:
This stuff is still true. I observed the same behavior and just explained it in different words

This pr was created before the commit, that's why it led to the merge conflict

I found that comment a bit confusing and decided to choose a different version